BOUNTY.md

```diff
@@ -0,0 +1,115 @@
+3. The GraphQL.NET team works with developers to clarify and refine their proposals.
+4. A proposal is accepted. The developer(s) responsible for the proposal work to implement the feature or
+   bug fix according to their proposal.
+    * Bounty status changed to **In Progress**
+5. Pull requests are submitted to implement the proposal by the developers.
+    * Bounty status changed to **Under Review**
+6. Pull requests are reviewed by project maintainers, and the pull request developers make any changes to the
+   pull requests accordingly.
+7. Pull requests are merged into the master or develop branch.
+8. Developer(s) submit invoice to the [GraphQL.NET Open Collective](https://opencollective.com/graphql-net)
+   for bounty amount, confirmed by GraphQL.NET team members.
+    * Bounty status changed to **Claimed**
+9. Payment disbursed to developer(s) by the Open Source Collective from GraphQL.NET funds.
+
+
+## How to Claim a Bounty
+
+We've created a process to ensure multiple contributors aren’t competing on the same project, and to make sure
+work is properly merged into GraphQL.NET once completed.
+
+1. Read the Request For Proposal document completely.
+2. If you want to move forward, submit an proposal to the GraphQL.NET team on your suggested implementation by
+   replying to the RFP posting. Be sure to include as many details as requested by the spec, and be sure to address
+   each of the completion criteria. For proposals that include UI changes, mockups are strongly encouraged.
+3. Work with the GraphQL.NET team to finalize plans for the best implementation. Proposals will be judged on their
+   maintainability, design quality, and adherence to spec. Note that other developers may submit competing proposals
+   at this time as well.
+4. If your proposal is accepted, you may begin code implementation that adheres to your proposal.
+5. Submit code as a draft pull request in the appropriate repository so GraphQL.NET community leaders can track progress.
+6. Once code is complete, remove draft status and notify GraphQL.NET maintainers.
+7. Update code with feedback from core GraphQL.NET contributors as needed.
+8. Wait for your code to be merged into the `master` or `develop` branch.
+9. [Submit an expense through Open Collective](https://opencollective.com/graphql-net/expenses/new) for the amount of the bounty.
+
+
+## Collaboration with GraphQL.NET team
+
+Collaboration with the GraphQL.NET team is crucial to having your code pulled into GraphQL.NET. During the proposal process,
+clarify any questions you or your team may have as early as possible. During development, use the pull request commenting
+feature.
+
+
+## Deadlines
+
+If your team does not show code commits or interaction for two weeks at a time as part of an accepted feature bounty, then
+the bounty will be released for another team to work on. If the task remains in the **Accepting Proposals** state, and the
```

Comment:
> will be released for another team to work on

So, `In Progress` -> `Accepting Proposals` transition here? If so that should be mentioned in Bounty Process Flowchart section, right?